src/components/ui/PopOver/index.tsx

```diff
@@ -156,7 +159,7 @@ export const PopOver: FC<PopOverProps> = ({
             {({ ref, style, placement }) => (
               <StyledPopOverMenu
                 data-placement={placement}
-                onClick={(e: any) => closePopover(e)}
+                onClick={(e: any) => {if (canToggle) {closePopover(e);}}}
```

Comment:
Can you break this out into a multi-line format to be consistent with the code style? Both places it's used.

Reply:
Will remove this prop.